### PR TITLE
Feature/2501 GP media library unit tests

### PR DESCRIPTION
**Pull request by sagarsdeshmukh**
Test cases added for -
- Add image in page from GP media library
- Select featured image for page from GP media library
- Select image in blocks plugin from GP media library

### CODE REVIEW

File: tests/p4/GPMLAddMedia.php [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+		$add->click();
+
+		// Validate Media modal window is shown.
+		$this->assertContains(
+			'GPI Media Library',
+			$this->webDriver->findElement( By::className('media-frame-title' ) )->getText()
+		);
+
+		// Wait for media content to load.
+		usleep(10000000);
```
Comment (kirdia) [inline tests/p4/GPMLAddMedia.php]: @sagarsdeshmukh is this wait necessary to switch frames/tabs?

File: tests/p4/GPMLAddMedia.php [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+		$ml_image_name = $ml_image_name[0];
+
+		$img = $this->webDriver->findElement( By::cssSelector( 'li.attachment:first-child' ) );
+		$img->click();
+		$this->webDriver->findElement( By::className( 'ml-button-insert' ) )->click();
+
+		// Switch back to default content.
+		$this->webDriver->switchTo()->defaultContent();
+
+		usleep(10000000);
```
Comment (kirdia) [inline tests/p4/GPMLAddMedia.php]: For such long waits I think it is better to add certain conditions that have to be matched in order to continue execution.
For example for this we can try if any ajax request is still pending.
```
        $driver->wait(30, 2000)->until(
            function () use ($driver) {
                return $driver->executeScript("return jQuery.active == 0;");
            }
        );
```

**Comment by angtheod** [comment]
The branch name and PR title is kind of misleading, since these are integration tests (not unit tests).

**Comment by sagarsdeshmukh** [comment]
To rename the branch, closed this PR and added a new one as below -
https://github.com/greenpeace/planet4-selenium-tests/pull/14

**Pull request closed without merging**